pkgs/by-name/su/supernovas/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  which,
+}:
+stdenv.mkDerivation rec {
+  pname = "supernovas";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "smithsonian";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-NkTcqsys+sZXXkkW8m0mT15uvSNfjkDWWsnxySermdY=";
+  };
```

tag and hash are preferred

```suggestion
  src = fetchFromGitHub {
    owner = "smithsonian";
    repo = "supernovas";
    tag = "v${version}";
    hash = "sha256-NkTcqsys+sZXXkkW8m0mT15uvSNfjkDWWsnxySermdY=";
  };
```